Documentation/policy/intro.rst

```diff
@@ -45,18 +47,34 @@ daemon:
 
     $ cilium-agent --enable-policy={default,always,never} [...]
 
+If there is only a specific endpoint that you want to put into a particular
+enforcement mode, you can do that with the following command, using an ID
+from ``cilium endpoint list``:
+
+.. code:: bash
+
+    $ cilium endpoint config <ID> PolicyEnforcement={default,always,never}
```

Comment:
One concern that I have is we have issues like https://github.com/cilium/cilium/issues/2496 with things like this.

Reply:
Fair concern, I would say that we should document what the intended and desired behaviour is, and consider deviations from this as bugs. We're already tracking this one already, so I think we're part way there.

Personally I've used this option and it seemed to work the way I expected, but if we're strongly concerned about its usage then I'd be happy to drop the documentation for it and leave it as a "deliberately undocumented feature", where if you use it, YMMV.